packages/is-prop-valid/src/props.js

```diff
@@ -480,11 +480,7 @@ const props = {
   y2: true,
   yChannelSelector: true,
   z: true,
-  zoomAndPan: true,
-  // preact
-  for: true,
-  class: true,
-  autofocus: true
```

Comment:
`@emotion/is-prop-valid` is independent of `@emotion/styled` and I know that some libraries were actually using it outside of Emotion. So I would prefer keeping those in for now - we can always reconsider removing them in the next major version.

Reply:
Great point :)